src/UI/Internal/Tables/FiltersView.elm

```diff
@@ -177,23 +177,23 @@ headerAttrs isApplied =
                 ]
 
             else
-                [ Background.color Colors.gray.lightest
-                , Font.color Colors.primary.darkest
-                , Element.mouseOver
-                    [ Background.color Colors.primary.lightest ]
-                ]
-                    ++ Element.colorTransition 100
+                Background.color
+                    Colors.gray.lightest
+                    :: Font.color Colors.primary.darkest
+                    :: Element.mouseOver [ Background.color Colors.primary.lightest ]
+                    :: Element.colorTransition 100
     in
-    [ Primitives.roundedBorders contextSize
-    , Element.width Element.fill
-    , paddingXY
-    , Element.spacing 8
-    , Font.size textSize
-    , Font.semiBold
-    , Element.pointer
-    ]
-        ++ (ARIA.toElementAttributes <| ARIA.roleButton)
-        ++ workingTheme
+    Primitives.roundedBorders
+        contextSize
+        :: Element.width Element.fill
+        :: paddingXY
+        :: Element.spacing 8
+        :: Font.size textSize
+        :: Font.semiBold
+        :: Element.pointer
+        :: (ARIA.toElementAttributes ARIA.roleButton
+                ++ workingTheme
```

Comment:
Even here, `O(n=9)` has gone to `O(9)` :kangaroo: 

@segmentationfaulter can we do it so that:
*  `(([1, 2, 3] ++ [4, 5, 6]) ++ [7, 8, 9])` becomes `([7, 8, 9] :: 1 :: 2 :: 3 :: 4 :: 5 :: 6)`. In here `O(9) !== O(6)`
*  `([1, 2, 3] ++ ([4, 5, 6] ++ [7, 8, 9]))` continues as it's? In here `O(6) === O(6)`